src/Form/EventCommandType.php

```diff
@@ -33,6 +38,26 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 },
             ])
         ;
+
+        if ($options['coalition']) {
+            $builder
+                ->add('coalition', EntityType::class, [
+                    'constraints' => [new NotBlank()],
+                    'class' => Coalition::class,
+                    'choice_label' => 'name',
+                    'query_builder' => function (CoalitionRepository $cr) {
+                        return $cr->findEnabled();
+                    },
+                ])
+                ->remove('capacity')
```

Comment:
what about creating a new FormType `CoalitionEventType` which will use this one as a parent FormType?